src/services/shell.service.js

```diff
@@ -12,3 +13,14 @@ export const openProjectInFolder = (project: Project) =>
 
 export const openProjectInEditor = (project: Project) =>
   launchEditor(project.path, 1, 1);
+
+export const checkIfNodeIsAvailable = () =>
```

Comment:
I think the name of this function implies it returns a boolean, not a version. Maybe it should be `getNodeJsVersionIfAvailable`?

We could also rename it `getNodeJsVersion`, update it so that it always `resolve`s instead of `reject`ing, and then handle it like this (assuming we go the async/await route):

```js
const nodeVersion = await getNodeVersion();

if (nodeVersion) {
  // success case
} else {
  // error case
}
```

I'm not picky on that last idea though